packages/components/inputs/select-utils/src/create-select-styles.ts

```diff
@@ -94,13 +94,17 @@ type TState = {
 
 type TDesignTokenName = keyof typeof designTokens;
 
+const upperFirst = (str: string) => str.charAt(0).toUpperCase() + str.slice(1);
```

Comment:
Perhaps we can use `upperFirst` from lodash, since it's already a dependency in this package

Reply:
I checked the main `package.json` file and didn't see that dependency so I assumed we were not using it in this repository 🤦 

Anyway, with the new approach we don't need that utility anymore.